fennel/testing/execute_aggregation.py

```diff
@@ -326,47 +329,240 @@ def get_val(self) -> List:
         return self.vals[min(len(self.vals) - 1, max(index - 1, 0))]
 
 
-def get_aggregated_df(
+def get_timestamps_for_hopping_window(
```

Comment:
Can we add tests for these functions ( get_timestamps_* ) too ?

Reply:
added